### PR TITLE
Big changes

### PR DESCRIPTION
**Pull request by Oreo4102004**
-> added background

-> level_tools , (platform , button , flag)

-> uncomment the things in game.py for a demo usage of them

### CODE REVIEW

File: src/level_tools.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+import pyglet
+from .player import Player
+
+
+class platform(pyglet.shapes.Rectangle):
```
Comment (AbooMinister25) [inline src/level_tools.py:5]: Why isn't the class name capitalized? Same goes for the `flag` class.

File: src/level_tools.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            self.x < other.x + other.width
+            and self.x + self.width > other.x
+            and self.y < other.y + other.height
+            and self.y + self.height > other.y
+            for self in flag_.parts
+        ):
+            return True
+
+
+class button:
```
Comment (AbooMinister25) [inline src/level_tools.py:69]: Same thing about class capitalization here

File: src/level_tools.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    def __init__(self, x, y, width, color, alignment="h"):
+        if alignment == "h":
+            height = 10
+        else:
+            height = width
+            width = 10
+
+        super().__init__(x, y, width=width, height=height, color=color)
+        self.on_platform = False
+        self.a = alignment
```
Comment (AbooMinister25) [inline src/level_tools.py:15]: Why not just make this `self.alignment`? It would pose better for readability

**Review by AbooMinister25 — CHANGES_REQUESTED** [review]

File: src/game.py [hunk 4]
```diff
@@ -1,14 +1,17 @@
 import pyglet
+from .level_tools import Player  # , platform, flag, button
+from .constants import WIDTH, HEIGHT
 
-from .constants import HEIGHT, WIDTH
-from .player import Player
+bg = pyglet.resource.image("bg.png")
```
Comment (brad90four) [inline src/game.py:5]: Could this PR include the image too?

**Review by brad90four — CHANGES_REQUESTED** [review]

**Review by brad90four — APPROVED** [review]
LGTM, loads of awesome work on this

**Review by Kronifer — APPROVED** [review]
Lgtm

**Pull request merged**